atomic_reactor/build.py

```diff
@@ -183,6 +185,31 @@ def set_df_path(self, path):
         for image in dfp.parent_images:
             self.parent_images[image] = None
 
+        # validate user has not specified COPY --from=image
+        builders = []
+        for stmt in dfp.structure:
+            if stmt['instruction'] == 'FROM':
+                # extract "bar" from "foo as bar" and record as build stage
+                match = re.search(r'\S+ \s+  as  \s+ (\S+)', stmt['value'], re.I | re.X)
```

Comment:
With `re.X` the literal whitespaces are chopped off?

Reply:
Yes, they are: "Whitespace within the pattern is ignored": https://docs.python.org/3/library/re.html#re.X

Neat!